### ISSUE TITLE
Registration Site Broken?

### ISSUE THREAD

**Issue by drkpie** [kochd/electricsheep-hd-client#22]
Can't seem to register, I go to submit and it leads to an internal server error message.

**Comment by kochd** [comment]
I don't exactly know what you mean by sketchy? I don't use sketchy Google browsers :).
Its probably due to the self signed certificate.

31. Januar 2019 13:28, "drkpie" <notifications@github.com (mailto:notifications@github.com?to=%22drkpie%22%20<notifications@github.com>)> schrieb:
	Oh I also don't want to open a seperate issue, so I hope you get comment notifications too. It seems Chrome likes to flag the registration site as sketchy hope you can figure out what's going there

**Comment by drkpie** [comment]
Yes that is probably the exact cause. By sketchy I mean there's the warning
screen before I get to the site.

**Comment by drkpie** [comment]
Alright seems the warning screen is gone so I figure you may have fixed that issue, site won't use https though. http://puu.sh/CFhbM/1ff48965fe.png Thanks for the quick responses and fixes though man!

**Comment by kochd** [comment]
no actually i did nothing about that

**Comment by drkpie** [comment]
I guess Chrome just likes to be silly lol.

**Comment by drkpie** [comment]
Oh, hey man, I know the software is still in alpha/beta and everything, but
I'm finding that it likes to cause a BSOD. Didn't get the chance to write
down the error code, but 100% it's not my desktop or an overclock.

**Comment by kochd** [comment]
No one ever reported something like this and there are people who ran the client for many hours on windows. Therefore i find it very unlikely that its actually related to this software rather than broken hardware or buggy operating system. also there is no sane reason why the software should cause the os kernel to panic.

2. Februar 2019 18:15, "drkpie" <notifications@github.com (mailto:notifications@github.com?to=%22drkpie%22%20<notifications@github.com>)> schrieb:
Oh, hey man, I know the software is still in alpha/beta and everything, but
I'm finding that it likes to cause a BSOD. Didn't get the chance to write
down the error code, but 100% it's not my desktop or an overclock.

**Comment by drkpie** [comment]
Most likely latest win updates fucking with things, thinking about it.
